daemon/api.go

```diff
@@ -1672,6 +1675,66 @@ type createResponseData struct {
 	SSHKeyCount int `json:"ssh-key-count"`
 }
 
+var userLookup = user.Lookup
+
+func addAuthUser(st *state.State, username string) error {
+	// create a new (local) macaroon
+	rootKey := make([]byte, 32)
+	if _, err := rand.Read(rootKey); err != nil {
+		return fmt.Errorf("cannot get random bytes: %s", err)
+	}
+	// FIXME: is this a proper use of ID and Location?
+	maca, err := macaroon.New(rootKey, username, "local-user")
+	if err != nil {
+		return fmt.Errorf("cannot create macaroon for the local user: %s", err)
+	}
+	macaB, err := maca.MarshalBinary()
```

Comment:
FYI, maybe you can use the same serialization format used by the store?
See store/auth.go, MacaroonSerialize.